doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-authoring-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='data-sets-authoring-con_{context}']
+= Data sets authoring
+
+A data set is a collection of related sets of information. It can be stored in many ways, such as in a database, in a Microsoft Excel file, or in memory. A data set definition instructs {CENTRAL} methods to access, read, and parse a data set. {CENTRAL} does not store data. It enables you to define access to a data set regardless of where the data is stored.
+
+For example, if data is stored in a database, a valid data set could contain the entire database or a subset of the database as a result of an SQL query. In both cases, the data is used as input for the reporting components of {CENTRAL} which then displays the information.
```

Comment:
Per style guidelines, "could" should not be used in the present tense to mean "might" or "can". Also, I think that the comma in the second sentence in not needed Suggest: 

For example, if data is stored in a database, a valid data set might contain the entire database or a subset of the database as a result of a SQL query. In both cases the data is used as input for the reporting components of {CENTRAL} which then displays the information.

Reply:
You are correct! 
I have removed "could" from the sentence.